GlottisFactory.cpp

```diff
@@ -0,0 +1,79 @@
+// ****************************************************************************
+// This file is part of VocalTractLab.
+// Copyright (C) 2020, Peter Birkholz, Dresden, Germany
+// www.vocaltractlab.de
+// author: Simon Stone
```

Comment:
I honestly don't know what the purpose of this preamble is in an open-source project and I only included it because the other files have it. What does copyright even mean in this context? What if an author revoked the permission to use the code they wrote? 

We should probably ask Peter how he wants this to be handled. I personally would not mind if we didn't have them at all. The VCS tracks individual contributions anyway and much more granular than a structured comment ever could.

Reply:
I think this preamble is there because the GPL-3.0 License suggests to put it there.
However, I also think it is rather non-standard, I mean the License is included anyway.

Reply:
Yeah. Technically copyright and license are different concerns, though. But I don't know and I don't really mind either way.